### DISCUSSION TITLE
A questin about github studentpack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22820] [New to GitHub]
<p>Hey, I’m a Surya from India, and I recently yesterday I applied for your student plan I’m having a small questin in it due to the pandemic this year my school didn’t issue a  student id card and I don’t have an educational mail id so I submitted my report card of this year and i don’t have an id card with me I want to just confirm that this will affect muy application for the student pack :blush:</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>We’re not the team who reviews the application so I can’t advise on the specifics of the documentation required. I can tell you that your application will be dealt with in order, and you’ll be notified if you need to provide more information.</p>

**Comment by ghost** [comment, marked as answer]
<p>Thank you for your reply but I’m late here :wink:</p>
